src/index.js

```diff
@@ -54,6 +56,7 @@ class PubsubPeerDiscovery extends Emittery {
     }
 
     this.removeListener = this.off.bind(this)
+    this.removeAllListeners = this.clearListeners.bind(this)
```

Comment:
What's using this? If this is being used we likely need to get more explicit about what event api peer-discovery modules should be exposing.

Reply:
It is being used fot the tests: https://github.com/libp2p/js-libp2p-interfaces/blob/v0.3.0/src/peer-discovery/tests/index.js#L25

I agree that we should be explicit about it.